src/extension/app/plugins/reload/reload.js

```diff
@@ -36,6 +37,7 @@ export function createReloadPlugin(appStore) {
         const res = await appStore.update();
         if (res) {
           appStore.showToast(appStore.i18n('reload_success'), 'positive');
+          appStore.fireEvent(EXTERNAL_EVENTS.RESOURCE_UPDATED);
```

Comment:
I'm wondering if we should start distinguishing between previewed from the editor vs reloaded?

One way we could do this to keep a single target for `updated` would be to add the environment to `sidekick` in the source as a suffix.

Reply:
Agreed, this change in logic should already be implemented in this PR.

When previewing we dispatch `EXTERNAL_EVENTS.RESOURCE_PREVIEWED`

When reloading we dispatch `EXTERNAL_EVENTS.RESOURCE_UPDATED` 

Previously we were dispatching [PREVIEWED](https://github.com/adobe/aem-sidekick/pull/258/files#diff-2232b3a8d76d8fdfb8c5d8f3d8fe25a9fc392b09dce95767f015a1c382061104L885) for both.